roles/taca/tasks/main.yml

```diff
@@ -0,0 +1,38 @@
+- name: Fetch TACA-ngi-pipeline from GitHub
+  git: repo={{ taca_ngi_repo }}
+       dest={{ taca_ngi_dest }}
+       version={{ taca_ngi_version }}
+       force=yes
+  tags: taca
+
+- name: Install TACA-ngi-pipeline requirements
+  shell: "{{ ngi_pipeline_venv }}/bin/pip install" -r "{{ taca_ngi_dest }}/requirements.txt"
+  tags: taca
+
+- name: Install TACA-ngi-pipeline
+  shell: cd "{{ taca_ngi_dest }}" && "{{ ngi_pipeline_venv }}/bin/pip" install .
+  tags: taca
+
+-name: Create TACA delivery configs directory
+  file: path="{{ ngi_resources }}/TACA" state=directory mode=g+s
+  tags: taca
```

Comment:
No. Since {{ ngi_pipeline_conf }} contains more or less static configuration whiles wheras taca uses its "conf" as an argument every time it is called, it'd be wrong to put it under conf. I contemplated making a seperate directory called taca_configs, but I felt it was a bit excessive.